charts/jindofs/values.yaml

```diff
@@ -30,6 +30,9 @@ properties:
   logDir: /mnt/disk1/bigboot/log
```

Comment:
Please also update the changelog and Chart.yaml with new version `0.8.5`? @frankleaf  Do you think it's fine?